ts/utils/biometrics.ts

```diff
@@ -121,3 +122,41 @@ export const isDevicePinSet = (): Promise<boolean> =>
       })
       .catch(_ => resolve(false));
   });
+
+export type BiometriActivationUserType =
+  | "ACTIVATED"
+  | "AUTH_FAILED"
+  | "PERMISSION_DENIED"
+  | "SENSOR_ERROR";
+
+export const mayUserActivateBiometric =
+  (): Promise<BiometriActivationUserType> =>
+    new Promise((resolve, reject) => {
+      getBiometricsType()
+        .then(value => {
+          if (value === "FACE_ID") {
+            FingerprintScanner.authenticate({
+              description: I18n.t(
+                "identification.biometric.popup.sensorDescription"
+              ),
+              fallbackEnabled: false
+            } as AuthenticateIOS)
+              .then(_ => resolve("ACTIVATED"))
+              .catch((err: Errors) => {
+                reject(handleErrorDuringBiometricActivation(err));
+              });
+          } else {
+            resolve("ACTIVATED");
+          }
```

Comment:
For this specific case, although it's possible from the code, it's practically impossible for "UNAVAILABLE" to be returned at that point. 
You can get to the authentication activation screen (where this function is used) only if a type of biometric authentication has been recognized as valid and available. 
However, in case it becomes "UNAVAILABLE" for some reason, after reaching the authentication activation screen, I thought it would still be useful to let the user activate the biometric (as happened so far). 
The only annoyance the user will receive, will be the system permission screen (only in case of FaceID) on first use.
In short, this is an extremely rare case and I think it is more correct to activate everything rather than launch an error.

Reply:
If you're thinking of a user who gets to the biometric activation screen and removes permission from the system settings, simply whatever choices he makes, will be irrelevant and the app won't consider biometrics from that point on. Furthermore, when it comes to a clean installation, it would not be possible to disable the setting from the system as it is not available at that point of onboarding yet.

Reply:
Okay then. Let's add some tests and their explanations too! 🚀